### ISSUE TITLE
JWT Configuration isn't taking hold

### ISSUE THREAD

**Issue by dannylamb** [Islandora-CLAW/CLAW#556]
When spinning up a new vagrant, JWT configuration is not set properly.  Both the Key and Secret are not available in their respective configuration panels.

Code that sets the config is here: https://github.com/Islandora-CLAW/claw_vagrant/blob/master/scripts/drupal.sh#L127

Code that checks for the token's existence is thwarted by PHPs terrible truth table.  Probably should stick to empty() instead of null checking here: https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Plugin/RulesAction/Broadcaster.php#L121

**Comment by dannylamb** [comment]
I stumbled upon this when trying to set up JWT AuthN for the OCR microservice I'm working on.  Shouldn't be major, but vagrant testing is always a bit of a pain.

**Comment by whikloj** [comment]
Sorry, when you say you see the key and JWT in their respective panels. But they are not configured? Because I rebuilt my vagrant today and it seemed to be configured. I can retry, but I want to make sure I understand what you're seeing.

**Comment by dannylamb** [comment]
@whikloj There's _always_ the possibility my build didn't come up right.  Let me try again and we'll talk if I've still got problems.

**Comment by dannylamb** [comment]
@whikloj Edited my late friday afternoon grammar fail.  Maybe it's more clear now.  Still spinning up a new box to duplicate as soon as I offload my latest round of work.

**Comment by dannylamb** [comment]
Threw up a PR so that if there is a config error, at least we're notified now.  OK, safe to destroy/up for me now.  I'll report back later.

**Comment by dannylamb** [comment]
Yeah, totally an environment issue on my end.  My original shell I spun up that environment in is gone, so I don't know how/why it failed.  But I just cycled my vagrant box and we're all good.  I'll still work on https://github.com/Islandora-CLAW/islandora/issues/48 so we can be notified of configuration error while also passing tests.

**Comment by jonathangreen** [comment]
Resolved with https://github.com/Islandora-CLAW/islandora/commit/cfcc16e3376bbafb08ced7047dd56e30251da44d
